scripts/googleAppScripts/README.MD

```diff
@@ -13,16 +13,43 @@ To use these scripts:
 1. Open the Google Sheet that is collecting form responses.
 2. Click on `Extensions > Apps Script` to open the Apps Script editor.
 3. Copy the contents of the `.gs` files into the editor.
-4. Set up your GitHub token in the script properties:
-    - In the Apps Script editor, click on `File > Project properties > Script properties`.
-    - Add a new property with the name `GITHUB_TOKEN` and paste your GitHub personal access token as the value.
-5. Save and deploy the script as a web app or bound script, depending on your needs.
-
-**⚠️ Security Note:** Do not hard-code your `GITHUB_TOKEN` in your script. Always store and access it using script properties to keep it secure.
 
 ## Authentication
 
-The `SheetToGithubIssue.gs` script requires a GitHub personal access token (PAT) with the appropriate permissions to create issues in your repository.
+The `SheetToGithubIssue.gs` script requires a GitHub personal access token (PAT) with the `public_repo` scope to create issues in the repository.
+
+### Storing Your GitHub Token
+
+- Store your GitHub token securely using the Apps Script's user properties:
+  ```javascript
+  function setGithubCredentials() {
+    setPersonalGithubToken();
+    setGithubRepositoryName();
+  }
+  
+  function setPersonalGithubToken() {
+    var personalGithubToken = 'YOUR_ACTUAL_GITHUB_TOKEN'; // Replace with your actual token
+    PropertiesService.getUserProperties().setProperty('GITHUB_TOKEN', personalGithubToken);
+  }
+  
+  function setGithubRepositoryName() {
+    var repositoryName = '4tals/LinksForIsrael'; // Replace with your repository name
+    PropertiesService.getUserProperties().setProperty('GITHUB_REPOSITORY', repositoryName);
+  }
+  ```
+- Execute the `setPersonalGithubToken` function once to store your token.
+
+### Retrieving Your GitHub Token and Repository Name in the Script
+
+- Retrieve your GitHub token and repository name within the script using:
```

Comment:
I don't understand the point of this section, this is already what the script is doing, anyone working on the script will see it...